components/sst_importer/src/sst_importer.rs

```diff
@@ -274,7 +286,24 @@ impl SSTImporter {
         let path = self.dir.get_import_path(name)?;
         let start = Instant::now();
         let sha256 = meta.get_sha256().to_vec();
-        let expected_sha256 = if sha256.len() > 0 { Some(sha256) } else { None };
+        let expected_sha256 = if !sha256.is_empty() {
+            Some(sha256)
+        } else {
+            None
+        };
+        if path.save.exists() {
+            return Ok(path.save);
+        }
+
+        let lock = self
+            .file_locks
+            .entry(name.to_string())
+            .or_insert(Mutex::new(()));
```

Comment:
```suggestion
            .or_default();
```

I am afraid that if we call `Mutex::new` each time, we may trigger some system calls each time too, which would be a little slow.🤔

Reply:
Since Mutex is meaningless. I 'll remove it.